addons/controls/README.md

```diff
@@ -0,0 +1,419 @@
+    </>
+  );
+};
+```
+
+And again, as above, this can be rewritten using [fully custom args](#fully-custom-args):
+
+```jsx
+export const Reflow = ({ count, label, ...args }) => (
+  <>{range(count).map((i) => <Button label={`${label} ${i}` {...args}} />)}</>
+);
+Reflow.args = { count: 3, label: 'reflow' };
+Reflow.argTypes = { count: { control: { type: 'range', min: 0, max: 20 } } };
+```
+
+## Configuration
+
+The controls addon can be configured in two ways:
+
+- Individual controls can be configured via [control annotation](#control-annotation),
+- The addon's appearance can be configured via [parameters](#parameters).
+
+### Control annotation
+
+As shown above in the [custom control args](#custom-controls-args) and [fully custom args](#fully-custom-args) sections, you can configure controls via a "control" annotation in the `argTypes` field of either a component or story.
+
+Here is the full list of available controls:
+
+| data type   | control type | description                                                    | options        |
+| ----------- | ------------ | -------------------------------------------------------------- | -------------- |
+| **array**   | array        | serialize array into a comma-separated string inside a textbox | separator      |
+| **boolean** | boolean      | checkbox input                                                 | -              |
+| **number**  | number       | a numberic text box input                                      | min, max, step |
+|             | range        | a range slider input                                           | min, max, step |
+| **object**  | object       | json editor text input                                         |                |
+| **options** | radio        | radio buttons input                                            |                |
+|             | inline-radio | inline radio buttons input                                     |                |
+|             | check        | multi-select checkbox input                                    |                |
+|             | inline-check | multi-select inline checkbox input                             |                |
+|             | select       | select dropdown input                                          |                |
+|             | multi-select | multi-select dropdown input                                    |                |
+| **string**  | text         | simple text input                                              | -              |
+|             | color        | color picker input that assumes strings are color values       | -              |
+|             | date         | date picker input                                              | -              |
+
+### Parameters
+
+Controls supports the following configuration parameters, either [globally or on a per-story basis](https://storybook.js.org/docs/basics/writing-stories/#parameters):
+
+- [expanded](#expanded)
+
+#### expanded
```

Comment:
Change the title to "Show property documentation"

This feature is amazing! TIL